thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -512,6 +512,8 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
 #ifdef _CONTAINERIZED
                     if (!globals->getPropBool("@_dafsStorage"))
                     {
+// JCSMORE->GH - I think this needs to change to pass in accessMode to get correct aliased plane
```

Comment:
Have you created a jira to come back to this?

Reply:
have now: https://track.hpccsystems.com/browse/HPCC-27518